src/Gridify/Builder/BaseQueryBuilder.cs

```diff
@@ -198,6 +198,14 @@ private static object AddIndexerNullCheck(IGMap<T> gMap, object query)
             {
                return BuildAlwaysFalseQuery(parameter);
             }
+         
+         if (value is DateTime dateTime)
+         {
+            if (GridifyGlobalConfiguration.DefaultDateTimeKind.HasValue)
```

Comment:
global configuration automatically gets injected into the mapper configuration in Gridify, so here it is better to use the mapper configuration instead.

Reply:
Good to know - did not know this. Fixing this.